src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,65 @@
+import './SearchForm.css';
+import Header from '../Header/Header';
+import { useState } from 'react';
+
+export default function SearchForm({
+  isLoggedIn,
+  onSignInClick,
+  handleLogout,
+  name,
+  handleRequestNews,
+}) {
+  const [searchValue, setSearchValue] = useState('');
+  const [canSearch, setCanSearch] = useState(false);
+
+  return (
+    <div className="news-search">
+      <Header
+        isLoggedIn={isLoggedIn}
+        onSignInClick={onSignInClick}
+        handleLogout={handleLogout}
+        name={name}
+      />
+      <form
+        className="seach-form"
+        onSubmit={(e) => {
```

Comment:
Good job that you add `onSubmit` to the `form` rather that `onClick` to the button

most students make mistakes here